rasa/core/actions/action.py

```diff
@@ -350,7 +350,7 @@ def _validate_action_result(self, result):
                 ". Failed to validate Action server response from API, "
                 "make sure your response from the Action endpoint is valid. "
                 "For more information about the format visit "
-                "{}/customactions/".format(DOCS_BASE_URL)
+                "{}/core/actions/".format(DOCS_BASE_URL)
```

Comment:
I think you want core/run-code-in-custom-actions/ ?

Reply:
It will be renamed docs/core/run-code-in-custom-actions.rst → docs/core/actions.rst

PR: https://github.com/RasaHQ/rasa/pull/3503/files

Reply:
ah ok 👍